xc7/tests/ddr/arty.sdc

```diff
@@ -0,0 +1,12 @@
+create_clock -period 5 sys4x_clk__main_clkout_buf1 -waveform {0.000 2.500}
```

Comment:
These clock net names are not found in the input net list.  @acomodi can you investigate how hard it is to fix VPR to accept the input net names?  Alternatively, can we modify the input verilog to avoid aliases on the clock nets so that the constraints can match the verilog?